source/mouseHandler.py

```diff
@@ -260,6 +260,10 @@ def pumpAll():
 
 def terminate():
 	global scrBmpObj, _shapeTimer
+	if winUser.getKeyState(winUser.VK_LBUTTON) & 32768:

```

Comment:
It would be great to replace this magic number `32768` with a named constant such as `MOUSE_BUTTON_DOWN`.
Alternatively, we could create an `IntEnum` in `keyboardHandler` and import it.

```py
class KeyState(IntEnum):
	TOGGLED = 1
	DOWN = 32768
```

Replacing other usages of these constants can be done in a separate issue and pull request

Reply:
Actually the enum should probably use the `0bx` binary format, as [MS docs defines](https://docs.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-getkeystate#return-value) these as

>* If the high-order bit is 1, the key is down; otherwise, it is up.
>* If the low-order bit is 1, the key is toggled. A key, such as the CAPS LOCK key, is toggled if it is turned on. The key is off and untoggled if the low-order bit is 0. A toggle key's indicator light (if any) on the keyboard will be on when the key is toggled, and off when the key is untoggled.

```py
class KeyState(IntEnum):
	"""Mirrors the return values in GetKeyState (also used in GetAsyncKeystate)
	https://docs.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-getkeystate#return-value
	"""
	TOGGLED = 0b1
	"""If the low-order bit is 1, the key is toggled.

	A key, such as the CAPS LOCK key, is toggled if it is turned on.
	The key is off and untoggled if the low-order bit is 0.
	A toggle key's indicator light (if any) on the keyboard will be on when the key is toggled,
	and off when the key is untoggled.
	"""
	DOWN = 0b1000000000000000
	"""If the high-order bit is 1, the key is down; otherwise, it is up."""
```